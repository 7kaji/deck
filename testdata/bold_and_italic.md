# Bold and italic

- _**Bold Italic** and Italic_
- __*Bold Italic* and Bold__

<!-- {"layout":"title-and-body"} -->

