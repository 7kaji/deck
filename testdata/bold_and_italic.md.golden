[
  {
    "layout": "",
    "titles": [
      "Title"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Bold Italic",
                "bold": true,
                "italic": true
              },
              {
                "value": " and Italic",
                "italic": true
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Bold Italic",
                "bold": true,
                "italic": true
              },
              {
                "value": " and Bold",
                "bold": true
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ]
  }
]